rest/config.go

```diff
@@ -651,7 +651,8 @@ func decodeAndSanitiseConfig(r io.Reader, config interface{}) (err error) {
 	}
 
 	b = base.ConvertBackQuotedStrings(b)
-
+	// Expand environment variables
```

Comment:
decodeAndSanitiseConfig isn't called by handlePutDbConfig, but it looks like it probably should.  Can you add that prior to config.setup() being called there?

Reply:
Also CreateDB.

Reply:
This change has been included in the latest commit. While working out this change the existing ReadJSONFromMIME has been refactored and renamed to ReadConfigJSON.